_locales/en/messages.json

```diff
@@ -242,15 +242,15 @@
   "extensionHasUpdated": {
     "message": "Scratch Addons has updated to v$1"
   },
-  "extensionUpdateInfo1_v1_33_2": {
-    "message": "Scratch just released an update that changes some of the colors around the website. If you'd like to update your settings to use these new colors, go back, or otherwise customize the colors, you can reset or change any themes from $1.",
+  "extensionUpdateInfo1_v1_34": {
+    "message": "New addon: \"arrow key incrementation in editor\", which lets you use the up and down arrow keys to increase or decrease number values in block inputs. Go to $1 to enable it.",
```

Comment:
One change I always wanted to make, but didn't have time, was to split the "Go to..." message into another string, since it is used frequently. Maybe now's the time.

Reply:
It's such a small string, I'm not sure if it's worth it

Reply:
I think there would be too many ways that "Go to $1 to enable it" could be changed slightly ("Go to $1 to change this" or something), where it's not really worth it.